### PR TITLE
Add import routes for tree species and benefits and add sql migrations

### PR DESCRIPTION
**Pull request by chromium-52**
ClickUp ticket: https://app.clickup.com/t/1dk46gh

## This PR

Add two tables, `TREE_SPECIES` and `TREE_BENEFITS`

- `TREE_SPECIES` stores data on a tree species's scientific name, common name, and [i-Tree](https://www.itreetools.org/) code (which is used to look up a tree's environmental benefits in the `TREE_BENEFITS` table explained below
- `TREE_BENEFITS` stores data on the actual environmental benefits (like the amounts of energy conserved and stormwater filtered) for a tree of a given tree species with a given diameter

Add two import routes, `importTreeSpecies` and `importTreeBenefits`, that can be used to populate the `TREE_SPECIES` and `TREE_BENEFITS` tables, respectively, with data from i-Tree

### CODE REVIEW

**Review by Tvcz — APPROVED** [review]
lgtm

File: api/src/main/java/com/codeforcommunity/dto/imports/TreeBenefitImport.java [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+  public void setHydroInterception(Double hydroInterception) {
+    this.hydroInterception = hydroInterception;
+  }
+
+  public Double getNaturalGas() {
+    return naturalGas;
+  }
+
+  public void setNaturalGas(Double naturalGas) {
```
Comment (huang0h) [inline api/src/main/java/com/codeforcommunity/dto/imports/TreeBenefitImport.java:187]: I really hope you didn't have to write all of these manually

File: api/src/main/java/com/codeforcommunity/dto/imports/ImportTreeSpeciesRequest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  @Override
+  public List<String> validateFields(String fieldPrefix) throws HandledException {
+    String newFieldPrefix = fieldPrefix + "importTreeSpeciesRequest.";
+
+    if (treeSpecies == null) {
+      return Collections.singletonList(newFieldPrefix + "treeSpecies");
+    }
+
+    return treeSpecies.stream()
+        .flatMap(ni -> ni.validateFields(newFieldPrefix).stream())
```
Comment (huang0h) [inline api/src/main/java/com/codeforcommunity/dto/imports/ImportTreeSpeciesRequest.java:35]: nit; same as above

**Review by huang0h — APPROVED** [review]
lgtm! 💯

File: api/src/main/java/com/codeforcommunity/dto/imports/TreeBenefitImport.java (hunk 1 above)
Comment (chromium-52) [reply api/src/main/java/com/codeforcommunity/dto/imports/TreeBenefitImport.java:187]: [Fortunately not](https://www.jetbrains.com/help/idea/generating-code.html#generate-getters-setters) but others required some copy-pasting :(

**Pull request merged**